packages/teams-js/src/public/pages.ts

```diff
@@ -523,6 +523,7 @@ export namespace pages {
      * @param handler The handler to invoke when the user presses their Team client's back button.
      */
     export function registerBackButtonHandler(handler: () => boolean): void {
+      ensureInitialized();
```

Comment:
Thanks for the quick fix! I'm a little surprised that a unit test didn't need to be updated because I would have expected a unit test to catch this. We should have unit tests for each API that fails if the library isn't initialized. Can you please add unit tests for this now that we have `ensureInitialized` added?

Reply:
Added in the latest commit.

Reply:
Thanks!